sdk/storage/azure-storage-blob/azure/storage/blob/_shared/uploads.py

```diff
@@ -460,6 +472,7 @@ def _upload_chunk(self, chunk_info):
             upload_stream_current=self.progress_total,
             **self.request_options
         )
+        chunk_end = chunk_info.offset + chunk_info.length - 1
```

Comment:
What was the point of moving this assignment?

Reply:
It was mostly organizational. The variable isn't needed until the last line so I just moved it after the upload call.